### DISCUSSION TITLE
Socket.io and Raspberry Pi

### DISCUSSION THREAD

**Discussion by elmcremotebridge** [socketio/socket.io#4618] [General]
I am looking for a small computer form factor. I would like to know if a Raspberry Pi 3 or 4 model B+ could handle socket.io for production environment. 

On average I can have 100 to 200 live connections that's basically a chat application. No video or audio just text message. Of course I need message between devices not to have long delay. 

Would a Raspberry Pi 3 or 4 model B+ work

**Comment by darrachequesne** [comment]
Hi! That's hard to tell, I guess you will have to benchmark it against your own use case.
